guide/onboarding/funding-a-wallet.md

```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+Alongside the payment request, explain to the user that a payment channel will be opened with the received amount and that a minimum amount should be sent. This amount is to cover the network fees for opening a new payment channel.
 
-Finally, new users can buy bitcoin directly from other bitcoin users such as friends, other users at a meetup, or intermediary services like [localbitcoins](https://localbitcoins.com/).
+For users who want to [manually open a channel]({{'/guide/onboarding/funding-a-wallet/#manual-channel-opening' | relative_url}}) this screen should be skippable.
 
 </div>
 
-{% include /tip/open.html label="Tip: Teach users how to transfer" icon="info" color="blue" %}
+{% include /tip/open.html label="⚡ Lightning service" icon="info" color="blue" %}
 
-Users funding new wallets from an exchange could potentially come into contact with unknown concepts, such as a bitcoin address, QR code, or transaction confirmations. This is an excellent opportunity to explain the exchange to personal wallet flow.
+With Lightning services, the user can fund their wallet directly with a Lightning payment saving time and costs.
 
 {% include /tip/close.html %}
 
-{% include fact/pros.html %}
-  - An easy and popular way to acquire bitcoin
-{% include fact/close.html %}
+### Network sync
 
+If your application uses a method like compact block filters (BIP157/158) to sync to the Bitcoin network, which may take some time to sync, communicate to the user that their funds will not appear until the sync is complete. You may even consider not presenting a payment request until this sync is complete to prevent user confusion that their funds have not shown up.
 
-{% include fact/cons.html %}
- - Managing taxes
- - Difficult to find businesses/individuals willing to pay in bitcoin
- - Hard to set up recurring payments
- - Not private
- {% include fact/close.html %}
+It's recommended to start any required syncing as soon as the user opens your application to limit wait times.
 
+## Funding the wallet
 
-### Redeem bitcoin using a gift card
+The next step for the user to take is sending bitcoin to the payment request.
 
-Gift cards allow users to buy and own small amounts of bitcoin without having to sign up for an exchange. These cards are sometimes available from local retailers.
+Once the bitcoin has been sent, indicate to the user when the wallet has received the transaction and communicate it's status. Once the initial transaction has been confirmed, it's recommended to open a payment channel for the user automatically. This avoids the user having to manually find a peer and open a channel.
 
-Users can get set up in minutes using this technique.
+Opening of a payment channel will require another on-chain transaction to be processed. If you choose to automatically open a channel for the user both the initial and channel opening transaction status can be indicated with one screen. This avoids exposing users to unnecessary complexity.
```

Comment:
> Opening of a payment channel will require another on-chain transaction to be processed.

How about zero-confirmation channels? So instead of `will require` I'd put `usually requires, unless you choose to take a bit more risk and offer zero-cons..`

Reply:
That will be covered in the Lightning services sub-page. However, there is a tip at the end of the `Funding a wallet` section that hints towards this.